spec/presenters/application_presenter_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+describe ApplicationPresenter do
+  subject(:application_presenter) { described_class.new }
+
+  describe "Interface" do
+    it { is_expected.to respond_to :checkins_date_range }
+  end
+
+  describe "checkins_date_range" do
+    before do
+      application_presenter.instance_variable_set("@params", from: nil, to: nil)
```

Comment:
it's better to get value from this variable via some method and stub this method instead of setting variable